config/default.yml

```diff
@@ -125,7 +126,7 @@ RSpec/AnyInstance:
   Description: Check that instances are not being stubbed globally.
   Enabled: true
   VersionAdded: '1.4'
-  StyleGuide: https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/AnyInstance
```

Comment:
Actually, hold on.
https://github.com/rubocop/rubocop-rails/pull/326/files#diff-3653213e3a153018a75cc7b2536b1d0217a6d8d75188d71cf45441812f789698R420 suggests that we can use `StyleGuide` links to point to `rspec.rubystyle.guide`, and use `Reference` for https://www.rubydoc.info links